docs/supported.md

```diff
@@ -69,6 +70,7 @@ Earlier and Later versions might be only partially supported, we don't fully kno
 | URL imports                                | ✅      |
 | build indicator<sup>7</sup>                | ❌      |
 | Turbopack-specific options<sup>8</sup>     | ❌      |
+| internationalized (i18n) routing           | 🔄      |
```

Comment:
yeah I was thinking we should do this too :slightly_smiling_face: 

Too bad that it is not covered in the [Next.js next.config.js documentation](https://nextjs.org/docs/api-reference/next.config.js/introduction), making the note at the top of the page slightly incorrect, could you update that one as well? :pray: 

for example like so:
```
> The following options have been gathered from the [Next.js next.config.js documentation](https://nextjs.org/docs/api-reference/next.config.js/introduction) alongside the [Advanced routing documentation](https://nextjs.org/docs/advanced-features/i18n-routing).
```
truthfully I think we'd need to check the whole advanced-features section then and add all the missing configs from there here (or in a separate "advanced configs" table), but that's definitely going to be time consuming so if you can't do that now I don't want it to necessarily block this PR for it and we can just go with a temporary fix like this instead.

Reply:
Yeah, that'd be a whole big task for another day 😅